docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+
+
+## Behavioral Models
+
+OpenROAD requires behavioral Verilog modules for a latch and a gated clock. These modules are used during synthesis and follow a specific naming convention.
+
+
+## Gated Clock
+
+To create this module, a gated clock standard cell is required. This standard cell is used to create the OpenROAD specific module. Following is the generic structure of the behavioral latch module:
+
+
+```
+# cells_clkgate.v
+##########################
+module OPENROAD_CLKGATE (CK, E, GCK);
+  input CK;
+  input  E;
+  output GCK;
+
+ `ifdef OPENROAD_CLKGATE
+    <clkgate_std_cell> latch (.CLK(CK), .GATE(E), .GCLK(GCK));
+ `else
+    assign GCK = CK;
+ `endif
+endmodule
+```
+
+
+
+## Latch
+
+Next is the generic latch. Once again, this requires a latch standard cell to be used. The structure of this module differs slightly from the `clkgate` module seen previously. Following is the generic structure of the behavioral latch module:
+
+		
+
+
+```
+# cells_latch.v
+########################
+module $_DLATCH_P_(input E, input D, output Q);
+     <d_latch_std_cell> _TECHMAP_REPLACE_ (
+       .D(D),
+       .G(E),
+       .Q(Q)
+       );
+endmodule
+
+module $_DLATCH_N_(input E, input D, output Q);
+     <d_latch_std_cell> _TECHMAP_REPLACE_ (
+       .D(D),
+       .GN(E),
+       .Q(Q)
+       );
+endmodule
+```
+
+
+This file contains two modules, `$_DLATCH_P_` and `$_DLATCH_N_`. Notice that `$_DLATCH_N_` has it’s gate input is negated making it enable on a low signal while `$_DLATCH_P_` has a non-negated gate input making it enable on a high signal.
+
+
+## FastRoute Configuration
+
+FastRoute is the tool used to global-route the design. FastRoute requires a Tcl file to set which routing layers will be used for signals, adjust routing layer resources, set which routing heuristic to use when routing, etc. It’s recommended to use the default `fastroute.tcl` due to its simplicity and effectiveness. Following is the default FastRoute configuration file.
+
+
+```
+# fastroute.tcl
+#####################
+set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.5
+
+set_routing_layers -signal $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER)
+```
+
+
+The first command, set_global_routing_layer_adjustment, adjusts the routing resources of the design. This effectively reduces the number of routing tracks that the global router assumes to exist. By setting it to the value of 0.5, this reduced the routing resources of all routing layers to 50% which can help with congestion and reduce the challenges for detail routing. The second command, set_routing_layers, sets the minimum and maximum routing layers for signal nets by using the -signal option.
```

Comment:
wrap `set_global_routing_layer_adjustment`, `set_routing_layers`, and `-signal` with single quotes